packages/teleport/src/Databases/Databases.story.tsx

```diff
@@ -23,7 +23,16 @@ export default {
   title: 'Teleport/Databases',
 };
 
-export const Loaded = () => <Databases {...props} />;
+export const Loaded = () => {
+  const [searchValue, setSearchValue] = useState('');
```

Comment:
Lets remove. I understand that you added it so it's easier to test this _one_ part of the component state _manually_ in the storybook. If so, add a unit-test instead and keep the stories primary for rendering testing.

Please do not use snapshot testing for adding these tests.